src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -34,7 +34,7 @@ export type SplitButtonProps = {
    * Style of the button.
    * @default 'default'
    */
-  styleType?: 'cta' | 'high-visibility' | 'default';
+  styleType?: 'cta' | 'high-visibility' | 'default' | 'borderless';
```

Comment:
You don't need to define this anymore, it is the same as the one from `ButtonProps` (so don't omit it)